### PR TITLE
Added theme:build command.

### PR DESCRIPTION
**Pull request by markdorison**
## Description
Added new theme:build command.

## Motivation / Context
Provide a way for various repos to build their theme via configuration.

## Testing Instructions / How This Has Been Tested
Tests should pass.

## Screenshots
image

### CODE REVIEW

**Review by mattdaniel — APPROVED** [review]
Nice 👍

File: src/Robo/Plugin/Commands/ThemeCommands.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+use Robo\Exception\TaskException;
+use Robo\Result;
+use Robo\Robo;
+use Robo\Tasks;
+
+/**
+ * Robo commands related to theme operations.
+ */
+class ThemeCommands extends Tasks
```
Comment (adamzimmermann) [inline src/Robo/Plugin/Commands/ThemeCommands.php:13]: Are we prototyping these for CHQ, then planning to move them to our shared repo? I haven't been following this very closely.

Comment (markdorison) [reply src/Robo/Plugin/Commands/ThemeCommands.php:13]: @adamzimmermann This is the public repo.

Comment (adamzimmermann) [reply src/Robo/Plugin/Commands/ThemeCommands.php:13]: 🤦🏼 I saw `chq` in the repo name and clearly quit reading after that.

**Pull request merged**